src/screens/LoadingAddresses.js

```diff
@@ -43,7 +43,7 @@ class LoadingAddresses extends React.Component {
   /**
    * canRedirect {boolean} set if can already redirect to the screen after loading
    */
-  state = { canRedirect: false };
+  state = { canRedirect: false, timeoutId: null };
```

Comment:
Missing this new parameter in the docstring

Reply:
Removed timeoutId from state.